drivers/overlay/ov_network.go

```diff
@@ -291,7 +377,19 @@ func (n *network) KeyPrefix() []string {
 }
 
 func (n *network) Value() []byte {
-	b, err := json.Marshal(n.vxlanID())
+	overlayNetmap := make(map[string]interface{})
+
+	s := n.subnets[0]
```

Comment:
This PR has the required infra to support multiple subnets. I had some issues in marshal/unmarshal of multiple subnets. So only that piece is missing. I will push it in a subsequent PR. Its not required right now.